app/client/cypress/integration/national.spec.js

```diff
@@ -0,0 +1,188 @@
+    ).should('exist');
+
+    // verify first tab contents are hidden
+    cy.findByTestId('hmw-national-water-conditions-tab')
+      .findByText(
+        'of our rivers and streams are healthy based on biological communities',
+      )
+      .should('not.be.visible');
+  });
+
+  it('The US Drinking Water Systems By Type pie chart is displayed', () => {
+    cy.findByText(/US Drinking Water Systems By Type/).should('be.visible');
+  });
+
+  it('The National Drinking Water table is displayed', () => {
+    cy.findByText('Submission Year Quarter', { timeout: 20000 })
+      .should('exist')
+      .and('be.visible');
+  });
+
+  it('Clicking a Glossary Term opens the Glossary panel', () => {
+    cy.get('span[data-term="Drinking Water Health-based Violations"]').click();
+    cy.get('#glossary')
+      .should('exist')
+      .and('be.visible');
+  });
+});
+
+describe('National Water Conditions tab', () => {
+  beforeEach(() => {
+    cy.visit('/national');
+  });
+
+  it('Clicking the “DISCLAIMER” button displays the disclaimer popup', () => {
+    const text = /^EPA has posted information through this/;
+
+    // verify opening the disclaimer
+    cy.findByText('Disclaimer').click();
+    cy.findByText(text).should('exist');
+
+    // verify closing the disclaimer
+    cy.findByTitle('Close disclaimer').click();
+    cy.findByText(text).should('not.exist');
+  });
+
+  it('From one of the sub-tabs, clicking “Expand All/Collapse All” expands/collapses the content', () => {
+    // verify that only "Expand All" is visible
+
+    const text = /^Biological condition tells us how healthy a waterbody is/;
+
+    cy.findAllByTestId('hmw-expand-collapse')
+      .filter(':visible')
+      .contains('Collapse All')
+      .should('not.exist');
+    cy.findAllByTestId('hmw-expand-collapse')
+      .filter(':visible')
+      .contains('Expand All')
+      .should('exist');
+
+    // Click "Expand All" and verify that "Collapse All" is now visible
+    cy.findAllByTestId('hmw-expand-collapse')
+      .filter(':visible')
+      .contains('Expand All')
+      .click();
+    cy.findAllByTestId('hmw-expand-collapse')
+      .filter(':visible')
+      .contains('Collapse All')
+      .should('exist');
+    cy.findAllByTestId('hmw-expand-collapse')
+      .filter(':visible')
+      .contains('Expand All')
+      .should('not.exist');
+
+    cy.findByText(text)
+      .filter(':visible')
+      .should('exist');
+
+    // Click "Collapse All" and verify that "Expand All" is now visible and accordion text is hidden
+    cy.findAllByTestId('hmw-expand-collapse')
+      .filter(':visible')
+      .contains('Collapse All')
+      .click();
+    cy.findAllByTestId('hmw-expand-collapse')
+      .filter(':visible')
+      .contains('Collapse All')
+      .should('not.exist');
+    cy.findAllByTestId('hmw-expand-collapse')
+      .filter(':visible')
+      .contains('Expand All')
+      .should('exist');
+    cy.findByText(text).should('not.exist');
```

Comment:
You could simplify this test quite a bit. In general only assert what you need to assert (in this case, that the accordion content is visible or not). A unit test would be better suited to make sure the accordion button text changes every time, but it's not difficult to roll that small assertion in as you're doing...just no need to assert both that "Collapse All" is shown and "Expand All" is not. Here's my suggestion on simplifying this, while still testing what you're wanting to test:

```suggestion
    const text = /^Biological condition tells us how healthy a waterbody is/;

    // verify that accordion is initially collapsed
    cy.findAllByText('Expand All').filter(':visible');
    cy.findByText(text).should('not.exist');

    // click button, and verify that accordion is expanded
    cy.findAllByText('Expand All')
      .filter(':visible')
      .click()
      .contains('Collapse All');
    cy.findByText(text).should('be.visible');

    // click button again, and verify that accordion is collapsed
    cy.findByText('Collapse All').click();
    cy.findByText(text).should('not.exist');
```